docs/developer-guide/ksqldb-reference/select-pull-query.md

```diff
@@ -15,7 +15,7 @@ Synopsis
 ```sql
 SELECT select_expr [, ...]
   FROM aggregate_table
-  WHERE key_column=key
+  WHERE key_column=key [, ...]

```

Comment:
This notation is confusing to me since the additional `key_column=key` clauses need to be separated by the `AND` keyword, not commas as this notation suggests. I'm not sure what the canonical way to denote this is. Maybe 
```suggestion
  WHERE key_column=key [AND ...]
```
?

Reply:
Ah good call! That's definitely better